trainingFunction.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+function dbPull():array{
+    $connectionString = 'mysql:host=db; dbname=training_diary';
+    $dbUsername = 'root';
+    $dbpassword = 'password';
+    $db = new PDO ($connectionString, $dbUsername, $dbpassword);
```

Comment:
there should be no space after PDO: `new PDO($connectionString, $dbUsername, $dbpassword);`